src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -28,39 +28,17 @@ namespace Tizen.NUI.BaseComponents
     /// <since_tizen> 6 </since_tizen>
     /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class Selector<T>
+    public class Selector<T> : ICollection<SelectorItem<T>>
```

Comment:
Could you check this warning message?
https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1710

Reply:
I suppressed the warning since the `Selector` is commonly used name in style.